### PR TITLE
Docwork

### PR DESCRIPTION
**Pull request by catherineheywood**
Changes to Java readme to align content with Python readme. Shift testing information into new configuration file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,76 +68,72 @@ Click 'set me up!' on https://bintray.com/gov-uk-notify/maven/notifications-java
...
 
+```java
+Notification notification = client.getNotification(status, notificationType);
+```
 
-## Testing
+Optional `status` can be one of:
 
-There is a main class that can be used to test the integration. It is also useful to read this class to see how to integrate with the notification client.
+* `sending`
```
Comment (servingUpAces) [inline README.md]: we should also mention that null is an acceptable value

File: README.md [hunk 2]
```diff
@@ -70,76 +68,72 @@ Click 'set me up!' on https://bintray.com/gov-uk-notify/maven/notifications-java
...
+* `delivered`
+* `permanent-failure`
+* `temporary-failure`
+* `technical-failure`
 
-On a command line build the project with Maven:
+Optional `notificationType` can be one of:
 
-`> mvn test`
+* `email`
```
Comment (servingUpAces) [inline README.md]: we should also mention that null is an acceptable value

File: README.md [hunk 3]
```diff
@@ -70,76 +68,72 @@ Click 'set me up!' on https://bintray.com/gov-uk-notify/maven/notifications-java
...
+
+You'll find your service ID on the **API integration** page.
+
+## Send a message
+
+Text message:
 
 ```java
-NotificationClient client = new NotificationClient(api_key, serviceId, "https://api.notifications.service.gov.uk");
+NotificationResponse response = client.sendSms(templateId, mobileNumber, personalisation);
```
Comment (quis) [inline README.md]: Checked with @servingUpAces, made the change in 39744b1b710b33d885289cc77d88abab0934a06b

File: README.md (hunk 1 above)
Comment (quis) [inline README.md]: Fixed in fd855c0b8d577acbb91bf2b9e7e9d46a675dd302

**Pull request merged**